README_GITFLEXIMOD.rst

```diff
@@ -107,11 +102,11 @@ Each entry specifies either a tag, a hash or a branch. To point to a new tag:
 
 #. Checkout the new component(s)::
 
-     ./manage_externals/checkout_externals
+     ./bin/git-fleximod
```

Comment:
./bin/git-fleximod update _component_

Reply:
ok if I use this syntax?
`./bin/git-fleximod update <component>`